objectRecognitionSource/build.gradle

```diff
@@ -243,22 +252,22 @@ dependencies {
             throw new Exception("Could not find opencv-${opencvVersion}.jar in '${System.env.OPENCV_LOC}'. Either add the jar to the "
                 + "folder specified in the environment variable OPENCV_LOC or remove the following files: ${depFileNames}")
         } else {
-            compile files("${System.env.OPENCV_LOC}/opencv-${opencvVersion}.jar")
+            implementation files("${System.env.OPENCV_LOC}/opencv-${opencvVersion}.jar")
         }
     }
     
     // Used for uploading documents (images) to VANTIQ with VANTIQ SDK
-    compile "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
+    implementation "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
 
     // Use JUnit test framework
-    testCompile 'junit:junit:4.12'
-    testCompile project(path:":extjsdk", configuration:"testArtifacts")
+    testImplementation 'junit:junit:4.12'
+    testImplementation project(path:":extjsdk", configuration:"testArtifacts")
 }
 
-task copyModels(type: Copy, dependsOn: [configurations.testRuntime, configurations.metaTest]) {
-    from configurations.testRuntime.find { it.name == "coco-${currentCocoVersion}.pb" },
-        configurations.testRuntime.find { it.name == "coco-${currentCocoVersion}.names" },
-        configurations.testRuntime.find { it.name == "coco-${currentCocoVersion}.meta" },
```

Comment:
These are just reference changes -- the configurations can't be resolved against this way, but the classpaths they create can.  So we go with the flow here....